TGServerService/PreactionHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace TGServerService
+{
+    // Some useful functions for triggering pre action events
+    partial class TGStationServer
+    {
+        const string EventFolder = "EventHandlers/";
+
+        string GetPath(string eventName)
```

Comment:
Instead of using strings you should use the event ids and call the handlers from the logging function

Reply:
e.g. You could have a file titled `35xx - Do something after repo cloned.bat`